### PR TITLE
Refactoring node status to new page

### PR DESCRIPTION
**Pull request by aritraghosh**
- Created a new page in architecture in node for the status
- Removed the current node status from concepts and moved it there

### CODE REVIEW

**Comment by aritraghosh** [comment]
Please add the refactor label to this PR

File: content/en/docs/reference/node/node-status.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+---
+title: Node Status
+weight: 80
+---
+
+
+## Node status
```
Comment (sftim) [inline content/en/docs/reference/node/node-status.md:7]:
```suggestion
## Node status fields
```

File: content/en/docs/reference/node/node-status.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+You can use `kubectl` to view a Node's status and other details:
+
+```shell
+kubectl describe node <insert-node-name-here>
+```
+
+Each section of the output is described below.
+
+### Addresses
```
Comment (sftim) [inline content/en/docs/reference/node/node-status.md:24]: After the move, this should be a second level heading (`##`).

File: content/en/docs/reference/node/node-status.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
+---
+title: Node Status
+weight: 80
+---
+
```
Comment (sftim) [inline content/en/docs/reference/node/node-status.md:5]: Please write a brief introduction. The introduction should link to https://kubernetes.io/docs/concepts/architecture/nodes/

File: content/en/docs/concepts/architecture/nodes.md [hunk 4]
```diff
@@ -163,131 +163,18 @@ that should run on the Node even if it is being drained of workload applications
...
-
-- The kubelet updates the node's `.status` either when there is change in status
-  or if there has been no update for a configured interval. The default interval
-  for `.status` updates to Nodes is 5 minutes, which is much longer than the 40
-  second default timeout for unreachable nodes.
-- The kubelet creates and then updates its Lease object every 10 seconds
-  (the default update interval). Lease updates occur independently from
-  updates to the Node's `.status`. If the Lease update fails, the kubelet retries,
-  using exponential backoff that starts at 200 milliseconds and capped at 7 seconds.
+See [Node Status](/docs/concepts/node/node-status) for more details
```
Comment (sftim) [inline content/en/docs/concepts/architecture/nodes.md:178]: Try this:
```suggestion

## Node heartbeats

Heartbeats, sent by Kubernetes nodes, help your cluster determine the
availability of each node, and to take action when failures are detected.

For nodes there are two forms of heartbeats:

* Updates to the [`.status`](/docs/concepts/node/node-status/) of a Node
* [Lease](/docs/concepts/architecture/leases/) objects
  within the `kube-node-lease`
  {{< glossary_tooltip term_id="namespace" text="namespace">}}.
  Each Node has an associated Lease object.

```

File: content/en/docs/reference/node/_index.md [hunk 5]
```diff
@@ -9,6 +9,8 @@ This section contains the following reference topics about nodes:
 * the kubelet's [checkpoint API](/docs/reference/node/kubelet-checkpoint-api/)
 * a list of [Articles on dockershim Removal and on Using CRI-compatible Runtimes](/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes/)
 
+* [Node Status information](/docs/reference/node/node-status)
```
Comment (sftim) [inline content/en/docs/reference/node/_index.md:12]:
```suggestion
* [Node `.status` information](/docs/reference/node/node-status/)
```

**Review by sftim — COMMENTED** [review]
Thanks

I recommend some tweaks.

File: content/en/docs/reference/node/node-status.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
+---
+title: Node Status
```
Comment (sftim) [inline content/en/docs/reference/node/node-status.md:2]:
```suggestion
content_type: reference
title: Node Status
```

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

**Comment by sftim** [comment]
Once this merges, we can tweak the new reference page further.

**Review by divya-mohan0209 — COMMENTED** [review]
Thanks for this refactor, @aritraghosh !
/approve

**Pull request merged**